cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredSortAgentScheduler.java

```diff
@@ -154,6 +154,28 @@ public void schedule(Agent agent, AgentExecution agentExecution, ExecutionInstru
   }
 
   @Override
+  public ClusteredSortAgentLock tryLock(Agent agent) {
+    // This score determines how much time the agent had left in the set it's currently in.
+    String releaseScore = agentScore(agent);
+    String acquireScore = acquireAgent(agent);
```

Comment:
not sure, but the current return value from the swap script isn't being (other than a not null check to ensure the call worked) so it could be changed to return the zrank score as the indicator of success